target/linux/mpc85xx/config-5.10

```diff
@@ -154,6 +147,11 @@ CONFIG_MTD_SPI_NOR=y
 # CONFIG_MVME2500 is not set
 CONFIG_NEED_PER_CPU_KM=y
 CONFIG_NEED_SG_DMA_LENGTH=y
+CONFIG_NET_DEVLINK=y
+CONFIG_NET_DSA=y
+CONFIG_NET_DSA_QCA8K=y
```

Comment:
The QCA8K and AT803X drivers can only be enabled on the p1010 subtarget. This saves some space in flash memory on other devices.

Reply:
Thanks. I moved it to subtarget.